cli/src/main.rs

```diff
@@ -257,7 +258,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
             // write to file
             if ADDRESS_REGEX.is_match(&cmd.target).unwrap() {
-                output_path.push_str(&format!("/{}/snapshot.csv", &cmd.target));
+                output_path.push_str(&format!(
+                    "/{}/{}/snapshot.csv",
+                    chain_id(&cmd.rpc_url).await.unwrap(),
+                    &cmd.target,
+                ));
             } else {
                 output_path.push_str("/local/snapshot.csv");
```

Comment:
should we include the chain id for this path as well?

Reply:
no, local won't have a chain ID!